### PR TITLE
Ballistics Screen Update

### PR DESCRIPTION
**Pull request by mbanks94**
- Added react-native-webview
- Added components for rendering images & webviews
- Updated navigation on Learn tab
- Update return of ballistics screen

make sure to npm install

### CODE REVIEW

**Comment by mbanks94** [comment]
Also I need to work on the webviews more but they work for the moment

**Comment by mbanks94** [comment]
-Added ui for pen chance, just placeholders for now

**Comment by jordykoppen** [comment]
@mbanks94 What was the issue again you wanted me to have a look at regarding the videos?

File: android/battleBuddyAndroid/src/screens/stacks/Learn/BallisticsScreen.js [hunk 1]
```diff
@@ -1,52 +1,94 @@
...
+
   return (
-    <ScrollableContainer>
-      {Data.map((post, index) => {
-        if (!post.title) {
+    <ScrollView>
+      {data.map((post, index) => {
+        if (post.maintitle) {
           return (
             <React.Fragment key={index}>
```
Comment (jordykoppen) [inline android/battleBuddyAndroid/src/screens/stacks/Learn/BallisticsScreen.js:44]: Can you extract these Fragments (including below) into their own component? Would make the component a lot cleaner and contained. Something like PostImage PostVideo etc.

File: android/battleBuddyAndroid/src/components/learn/Button.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+`;
+
+const ButtonText = styled.Text`
+  color: ${({theme}) => theme.colors.white};
+  font-size: 20;
+  text-align: center;
+`;
+
+const Button = ({title}) => {
+  const onPressHandler = (title) => {
```
Comment (jordykoppen) [inline android/battleBuddyAndroid/src/components/learn/Button.js:22]: I know this is just for testing purposes but maybe pass it as an onPress instead. So it's reusable

**Review by jordykoppen — CHANGES_REQUESTED** [review]
Besides my two comments, it looks really good! 
Thanks a lot, Michael :)

File: android/battleBuddyAndroid/src/screens/stacks/Learn/BallisticsScreen.js (hunk 1 above)
Comment (mbanks94) [reply android/battleBuddyAndroid/src/screens/stacks/Learn/BallisticsScreen.js:44]: Will do!

File: android/battleBuddyAndroid/src/components/learn/Button.js (hunk 2 above)
Comment (mbanks94) [reply android/battleBuddyAndroid/src/components/learn/Button.js:22]: I intended on doing that!

File: android/battleBuddyAndroid/src/components/learn/Image.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  resize-mode: cover;
+  align-self: center;
+  width: 400;
+  height: 250;
+`;
+
+const CustomImage = ({name, ...props}) => {
+  let img;
+  switch (name) {
+    case '0':
```
Comment (jordykoppen) [inline android/battleBuddyAndroid/src/components/learn/Image.js:15]: I'm not a big fan of hardcoding the assets in this way. Also, you're using the data in the test-data folder which needs to be ported to the locales jsons. Maybe you can determine image/video or other assets by including the asset into the actual object. 
Just some thoughts

File: android/battleBuddyAndroid/test-data/ballistics.js [hunk 4]
```diff
@@ -1,8 +1,8 @@
 const ballisticsdata = [
```
Comment (jordykoppen) [inline android/battleBuddyAndroid/test-data/ballistics.js:1]: As I mentioned in a comment above, these need to be inserted into the locale json instead.

**Review by jordykoppen — COMMENTED** [review]
A few remaining thins

**Comment by mbanks94** [comment]
@jordykoppen Thanks for the suggestions man! Ill get this revised asap

**Comment by jordykoppen** [comment]
@mbanks94 Any updates?

**Comment by jordykoppen** [comment]
Closing as I'm fixing these comments and making additions myself

**Pull request closed without merging**